build.sbt

```diff
@@ -686,7 +686,20 @@ lazy val `logging-utils` = project
   .settings(
     version := "0.1",
     libraryDependencies ++= Seq(
-      "org.netbeans.api" % "org-netbeans-modules-sampler" % "RELEASE130",
+      "org.netbeans.api" % "org-netbeans-modules-sampler" % "RELEASE130"
+      exclude("org.netbeans.api", "org-openide-loaders")
+      exclude("org.netbeans.api", "org-openide-nodes")
+//    exclude following when RELEASE140 is out:
+//      exclude("org.netbeans.api", "org-netbeans-api-progress-nb")
+//      exclude("org.netbeans.api", "org-netbeans-api-progress")
+//      exclude("org.netbeans.api", "org-openide-util-lookup")
+//      exclude("org.netbeans.api", "org-openide-util")
+//      exclude("org.netbeans.api", "org-openide-dialogs")
```

Comment:
According to the [release schedule](https://cwiki.apache.org/confluence/display/NETBEANS/Release+Schedule) NetBeans 14 shall be released by end of May. At that time the _sampler library_ `RELEASE14` is going to be uploaded to [Maven Central](https://repo1.maven.org/maven2/org/netbeans/api/org-netbeans-modules-sampler/) and we can use its JAR - just one JAR as it will not need any additional dependencies.

Or is there some other way to include a JAR file into our`sbt` build? Asking @radeusgd , @hubertp , @4e6 ... Or are we OK with current set dependencies, @4e6?

Reply:
I'm to merge it with the current dependencies, and upgrade to version `140` when it is released